example/cmd/device-simple/res/Simple-Driver.yaml

```diff
@@ -22,6 +22,10 @@ resources:
             - { operation: "get", object: "SwitchButton", property: "value", parameter: "Switch" }
         set:
             - { operation: "set", object: "SwitchButton", property: "value", parameter: "Switch" }
+    -
+        name: "Image"
+        get:
+          - { operation: "get", object: "SwitchButton", property: "value", parameter: "Image" }
```

Comment:
The deviceResources needs to add an "Image" with the proper type, not reusing SwitchButton

Reply:
@cloudxxx8 The Image endpoint is not independently taking any PUT to change the value, but is reusing SwitchButton.

So it reuses SwitchButton here to reflect that it is toggling the image based on the true/false value of the Switch

Reply:
If I update the images used to represent on vs off states of the switch, would that be sufficient?  Or do you prefer having this become more independent device resource?